<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserTesting</class>
 <widget class="QDialog" name="UserTesting">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserTesting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color : white;
background-color : rgb(49,48,43);

QLineEdit{
color: white;
background-color : rgb(38, 38, 36);
border-color: rgb(76, 76, 76);
}

QPushButton:disabled
{
  background-color: rgb(225,67,19) ;
  border: 2px solid #8f8f91;  
  border-radius: 6px;
 color: white ;
}

#InfoTab {
color: white;
background-color : rgb(49, 48, 43);
}

QFrame
{
border:1px solid white;
}

QLabel
{
border:none;
}

QPushButton:checked
{
  background-color: rgb(225,67,19) ;
  border: 2px solid #8f8f91;  
  border-radius: 6px;
 color: white ;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>681</width>
     <height>241</height>
    </rect>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit_Output">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_TASKS">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadiobutton::indicator::checked
{
	background-color:black;
} 

QRadioButton::indicator::unchecked
{
	background-color:white;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="radioTask1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>TASK 1</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioTask2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>TASK 2</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioTask3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>TASK 3</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_DEVICES">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>130</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QRadiobutton::indicator::checked
{
	background-color:black;
} 

QRadioButton::indicator::unchecked
{
	background-color:white;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="radioMouse">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>MOUSE</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioTouch">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>MULTITOUCH</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioLeap">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>LEAP</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnCounterBal">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Debug">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTest">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameButtons">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>161</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
border:1px solid white;
}

QLabel
{
border:none;
}

QPushButton:checked
{
  background-color: rgb(225,67,19) ;
  border: 2px solid #8f8f91;  
  border-radius: 6px;
 color: white ;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btnStart">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>START</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnStop">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>STOP</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDone">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>DONE</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>USER ID</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxUSERID">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="maximum">
      <number>27</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>PART</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBoxPARTProto">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QTimeEdit" name="timeEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>1998</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="maximumDateTime">
      <datetime>
       <hour>23</hour>
       <minute>59</minute>
       <second>59</second>
       <year>1998</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::HourSection</enum>
     </property>
     <property name="timeSpec">
      <enum>Qt::LocalTime</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>4</width>
     <height>13</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
